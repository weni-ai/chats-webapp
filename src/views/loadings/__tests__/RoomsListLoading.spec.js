import { mount } from '@vue/test-utils';
import RoomsListLoading from '../RoomsList.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(RoomsListLoading);
};

describe('RoomsListLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
