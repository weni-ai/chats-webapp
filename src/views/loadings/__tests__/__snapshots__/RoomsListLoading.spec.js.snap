// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`RoomsListLoading.vue > to match snapshot 1`] = `
"<div data-v-e8ef6379="" class="rooms-list-loading">
  <div data-v-e8ef6379="" class="rooms-list-loading__room-list">
    <header data-v-e8ef6379=""><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 22px;"> ‌ </span></span></header>
    <main data-v-e8ef6379="">
      <ul data-v-e8ef6379="" class="rooms-list-loading__room-group">
        <li data-v-e8ef6379="" class="rooms-list-loading__room"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 40px;"> ‌ </span></span>
          <div data-v-e8ef6379="" class="rooms-list-loading__room__infos"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 20px;"> ‌ </span></span>
        </li>
        <li data-v-e8ef6379="" class="rooms-list-loading__room"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 40px;"> ‌ </span></span>
          <div data-v-e8ef6379="" class="rooms-list-loading__room__infos"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 20px;"> ‌ </span></span>
        </li>
      </ul>
    </main>
  </div>
  <div data-v-e8ef6379="" class="rooms-list-loading__room-list">
    <header data-v-e8ef6379=""><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 22px;"> ‌ </span></span></header>
    <main data-v-e8ef6379="">
      <ul data-v-e8ef6379="" class="rooms-list-loading__room-group">
        <li data-v-e8ef6379="" class="rooms-list-loading__room"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 40px;"> ‌ </span></span>
          <div data-v-e8ef6379="" class="rooms-list-loading__room__infos"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 20px;"> ‌ </span></span>
        </li>
        <li data-v-e8ef6379="" class="rooms-list-loading__room"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 40px;"> ‌ </span></span>
          <div data-v-e8ef6379="" class="rooms-list-loading__room__infos"><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div><span data-v-e8ef6379=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite; height: 20px;"> ‌ </span></span>
        </li>
      </ul>
    </main>
  </div>
</div>"
`;
